api/api.go

```diff
@@ -424,6 +424,10 @@ func (a *API) Get(ctx context.Context, decrypt DecryptFunc, manifestAddr storage
 		}
 		mimeType = entry.ContentType
 		log.Debug("content lookup key", "key", contentAddr, "mimetype", mimeType)
+		// TODO: obtain Publisher from cli flag if no Publisher found in Manifest
```

Comment:
please see this comment

Reply:
this refers to
`the fallback publisher we discussed with @vojtechsimetka?`

Reply:
#2180